pkgs/servers/web-apps/hedgedoc/default.nix

```diff
@@ -84,19 +85,19 @@ in stdenv.mkDerivation {
   installPhase = ''
     runHook preInstall
 
-    mkdir -p $out/share/hedgedoc
-    cp -r bin $out
-    cp -r {app.js,lib,locales,node_modules,package.json,public} $out/share/hedgedoc
+    mkdir -p $out
+    cp -R {app.js,bin,lib,locales,node_modules,package.json,public} $out
 
     for bin in $out/bin/*; do
       wrapProgram $bin \
         --set NODE_ENV production \
-        --set NODE_PATH "$out/share/hedgedoc/lib/node_modules"
+        --set NODE_PATH "$out/node_modules" \
+        --set PATH "$PATH:${nodejs}/bin:${bash}/bin"
```

Comment:
nodejs was required becaus the shebang was not fixed. Which script wanted to use bash?

Reply:
`bin/setup`

Reply:
I decided that we are going to delete that script in #312495, so we don't need that anymore.